src/ts/Html5Metadata.ts

```diff
@@ -59,7 +59,7 @@ export class Html5Metadata implements Conviva.MetadataInterface {
 
   // Relying on HTTP user agent string parsing on the Conviva Platform.
   getDeviceCategory(): Conviva.Client.DeviceCategory {
-    return null;
+    return Conviva.Client.DeviceCategory.WEB;
```

Comment:
What if you use it on mobile browsers? (Mobile Safari, Chrome, ...) Same for chromecast?

Reply:
Hello, 
this is just a quick fix to set something (it is critical because it should not be empty). There must be of course some device-checks.

Reply:
Why not adding those now?